src/translator/response.h

```diff
@@ -54,11 +59,11 @@ struct Response {
   /// translated text and annotations of (sub-)words and sentences.
   AnnotatedText target;
 
-  /// -logprob of each word and negative log likelihood of sequence (sentence)
+  /// logprob of each word and  the total sequence (sentence)
   /// normalized by length, for each sentence processed by the translator.
   /// Indices correspond to ranges accessible through respective Annotation on
   /// source or target.
-  std::vector<Quality> qualityScores;
+  std::vector<SentenceQualityEstimate> qualityScores;
```

Comment:
A very small comment. It makes sense to use the consistent naming everywhere i.e. either `Estimate` or `Score`.

Reply:
Commit 6fbc1bf